src/bin/pg_autoctl/monitor_pg_init.c

```diff
@@ -45,6 +46,7 @@ GUC monitor_default_settings[] = {
 	{ "log_connections", "off" },
 	{ "log_disconnections", "off" },
 	{ "log_lock_waits", "on" },
+	{ "log_statement", "all" },
```

Comment:
Yeah we could review that and change it, it's been quite useful in the past to have it though. I agree that nowadays in case of failures we have too much logs to go through... but that still beats the alternative.